plugins/getty-images/src/GettyImagesToolControlBar.tsx

```diff
@@ -0,0 +1,208 @@
+/*
+  This file is part of an img.ly Software Development Kit.
+  Copyright (C) 2016-2021 img.ly GmbH <contact@img.ly>
+  All rights reserved.
+  Redistribution and use in source and binary forms, without
+  modification, are permitted provided that the following license agreement
+  is approved and a legal/financial contract was signed by the user.
+  The license agreement can be found under the following link:
+  https://www.photoeditorsdk.com/LICENSE.txt
+*/
+
+import React, { useEffect, useMemo, useRef, useState } from 'react';
+
+import {
+  CustomToolbar,
+  CustomToolProps,
+  SearchField,
+  useSetImage,
+} from 'photoeditorsdk';
+import {
+  QueryClient,
+  QueryClientProvider,
+  useInfiniteQuery,
+} from 'react-query';
+import { useDebounce } from 'use-debounce';
+
+import {
+  createAPIClient,
+  DisplaySizeName,
+  GettyImage,
+  SearchImagesParams,
+} from './api';
+import { GettyImagesCards } from './components/GettyImagesCards';
+import { getDisplaySize, gettyStore, hasNextPage } from './helpers';
+import { useIntersectionObserver } from './hooks/useIntersectionObserver';
+import { useToken } from './hooks/useToken';
+import { ErrorNames, OnError } from './types';
+import {
+  CardContainer,
+  ScrollContainer,
+  SearchDiv,
+  Spinner,
+  ToolbarWrapper,
+} from './components/Elements';
+
+interface Props {
+  client: ReturnType<typeof createAPIClient>;
+  language: CustomToolProps['language'];
+  onError: OnError;
+  searchParams?: SearchImagesParams;
```

Comment:
we should use `Partial` here, else `SearchImagesParams.phrase` is always required and would override the search field phrase

Reply:
Right, it should be skipped at all